README.md

```diff
@@ -1,217 +1,263 @@
 # vrs-python
 
-[![PyPI version](https://badge.fury.io/py/ga4gh.vrs.svg)](https://pypi.org/project/ga4gh.vrs)
-
-vrs-python provides Python language support for the [GA4GH Variation
+**VRS-Python** provides Python language support for the [GA4GH Variation
 Representation Specification
 (VRS)](https://github.com/ga4gh/vrs).
 
-This repository contains several related components:
+## Information
+
+[![license](https://img.shields.io/badge/license-Apache-green)](https://github.com/ga4gh/vrs-python/blob/main/LICENSE) [![binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ga4gh/vrs-python/main?labpath=notebooks)
+
+## Releases
+
+[![gitHub tag](https://img.shields.io/github/v/tag/ga4gh/vrs-python.svg)](https://github.com/ga4gh/vrs-python/releases) [![pypi](https://img.shields.io/pypi/v/ga4gh.vrs.svg)](https://pypi.org/project/ga4gh.vrs/)
+
+## Development
+
+ [![action status](https://github.com/ga4gh/vrs-python/actions/workflows/python-cqa.yml/badge.svg)](https://github.com/ga4gh/vrs-python/actions/workflows/python-cqa.yml) [![issues](https://img.shields.io/github/issues-raw/ga4gh/vrs-python.svg)](https://github.com/ga4gh/vrs-python/issues)
+[![GitHub Open Pull Requests](https://img.shields.io/github/issues-pr/ga4gh/vrs-python.svg)](https://github.com/ga4gh/vrs-python/pull/) [![GitHub license](https://img.shields.io/github/contributors/ga4gh/vrs-python.svg)](https://github.com/ga4gh/vrs-python/graphs/contributors/) [![GitHub stars](https://img.shields.io/github/stars/ga4gh/vrs-python.svg?style=social&label=Stars)](https://github.com/ga4gh/vrs-python/stargazers) [![GitHub forks](https://img.shields.io/github/forks/ga4gh/vrs-python.svg?style=social&label=Forks)](https://github.com/ga4gh/vrs-python/network)
 
-- **ga4gh.core package** Python language support for certain nascent standards
-  in GA4GH. Eventually, this package should be moved to a distinct repo.
+## Features
```

Comment:
@korikuzma I think the Features section should not be broken down by module. But instead it should be a bullet list of separate features (see [hgvs read.me](https://github.com/biocommons/hgvs?tab=readme-ov-file#features)).  I think it is harder to discern the specific *features* as it feels more technical to go through modules right out of the gate.

Reply:
@larrybabb should we be distinguishing between VRS Python features (ie the translator and annotator) and VRS features (a nonambiguous description of variation)? Or maybe bullet point one is like "Pydantic implementation of VRS models, which enables <...>" and then the next bullet points are the additional tools provided in the package?

Reply:
Updated in last commit. Should I include dataproxy or nah?